ddh-intro.md

```diff
@@ -37,7 +39,7 @@ We welcome new contributors to dive in and improve live Instant Answers. It's a
 - Tackle some of the "low-hanging fruit" and, "high priority" issues on our [Live Issues Page](https://duck.co/ia/dev/issues?tag=lowhangingfruit)
 - Help other DuckDuckHackers with their [Instant Answers in development](https://duck.co/ia/dev/pipeline)
 - Suggest improvements & file bugs for [live Instant Answers](https://duck.co/ia) through the, "Create Issue" button on each IA page
-
+- Join the discussion in our [forum](https://forum.duckduckhack.com/tags/c/programming/needs-input) about how we can improve Instant Answers for programmers. 
```

Comment:
If this section is for learning, then maybe pointing to specific examples of good brainstorming like the MDN or JS threads in the forum.